app/email/confirm-email/page.tsx

```diff
@@ -1,8 +1,10 @@
-import {redirect} from 'next/navigation';
+import { redirect } from 'next/navigation';
+
+export const dynamic = 'force-dynamic'
 
 async function verifyEmail(userId: string, token: string) {
   // Verify the email address
-  const response = await fetch(`${process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT}/verify-email`, {
+  const response = await fetch(`${process.env.NEXT_PUBLIC_SERVER_ENDPOINT}/verify-email`, {
```

Comment:
Just wondering if the different environment variable is intentional. If so all looks good to me.

Reply:
Thanks @andrewebdev. Yea, I noticed that both of those env variables had the same value, so I switched the instances of `NEXT_PUBLIC_GRAPHQL_ENDPOINT` to use `NEXT_PUBLIC_SERVER_ENDPOINT`